src/OmniSharp.Abstractions/OmniSharp.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
     <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
we could have all the libs as netstandard2.0 and just keep the entry projects net461

Reply:
:+1: on this!

Reply:
be careful with this. There are known issues with netstandard2.0. I'd recommend sticking with net461 if there isn't a reason to switch.

Reply:
technically there is no real reason - other than maybe when we publish Nuget packages there might be easier to consume as `netstandard2.0`. That said I think the only reason anyone would consume OmniSharp packages would be to build things for OmniSharp so it wouldn't matter anyway